test/jpypetest/test_boxed.py

```diff
@@ -169,8 +169,9 @@ def testPrivitiveToBoxed(self):
         java.lang.Double(JDouble(0))
 
     def testBooleanBad(self):
-        with self.assertRaises(TypeError):
-            java.lang.Boolean(tuple())
+        # with self.assertRaises(TypeError):
+        # This test was wrong.   'bool(tuple()) == False'
+        self.assertFalse(java.lang.Boolean(tuple()))
```

Comment:
The same thing applies to empty lists, dicts and sets, doesn't it?

Reply:
Yes.  I am not sure I like this behavior too much as it makes JBool and java.lang.Boolean very permissive.  But since that technically is Python rules regarding apply bool to complex object , it the behavior that the user expected on an explicit cast.  I guess I should test the implicit case as well.